### PR TITLE
Migrate TABULAR Worker Monitoring Reports to Bootstrap 3

### PR DESCRIPTION
**Pull request by biyeun**
putting this on staging

@dimagi/product 
@dannyroberts @benrudolph @czue 
cc @kaapstorm 

screen shot 2016-01-15 at 1 31 53 pm
screen shot 2016-01-15 at 1 31 43 pm
screen shot 2016-01-15 at 1 31 31 pm
screen shot 2016-01-15 at 1 31 15 pm
screen shot 2016-01-15 at 1 31 07 pm
screen shot 2016-01-15 at 1 30 57 pm
screen shot 2016-01-15 at 1 30 49 pm

### CODE REVIEW

File: corehq/apps/reports/dont_use/fields.py [hunk 1]
```diff
@@ -45,10 +53,16 @@ def update_context(self):
         """
         pass
 
+    def get_bootstrap_template(self):
+        if self.is_bootstrap3:
+            return self.template.replace('/bootstrap2/','/bootstrap3/')
```
Comment (dimagimon) [inline corehq/apps/reports/dont_use/fields.py]: E231 missing whitespace after ','

File: corehq/apps/reports/filters/base.py [hunk 2]
```diff
@@ -65,12 +75,17 @@ def render(self):
...
-        return render_to_string(self.template, self.context)
+        return render_to_string(self.get_bootstrap_template(), self.context)
 
     @classmethod
     def get_value(cls, request, domain):
         return request.GET.get(cls.slug)
 
+    def get_bootstrap_template(self):
+        if self.is_bootstrap3:
+            return self.template.replace('/bootstrap2/','/bootstrap3/')
```
Comment (dimagimon) [inline corehq/apps/reports/filters/base.py]: E231 missing whitespace after ','

**Comment by czue** [comment]
whoa, awesome!

**Comment by amsagoff** [comment]
:open_mouth:

**Comment by kaapstorm** [comment]
:lollipop: Sweet!

**Comment by dannyroberts** [comment]
```
======================================================================
ERROR: testBasicFunctionality (corehq.apps.reports.tests.test_cache.ReportCacheTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_cache.py", line 56, in testBasicFunctionality
    v1 = report.v1()
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/quickcache.py", line 308, in inner
    return helper(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/cache.py", line 16, in __call__
    self.prefix = '{}{}'.format(self.prefix, '/b3' if report.is_bootstrap3 else '')
AttributeError: 'MockReport' object has no attribute 'is_bootstrap3'
```

**Comment by biyeun** [comment]
closing for now until https://github.com/dimagi/commcare-hq/pull/9986 gets merged

**Pull request closed without merging**